_common/README.md

```diff
@@ -14,45 +14,44 @@ Here are the syntaxes of the Alerts for Keyman websites. There are 5 types of GF
 
 ## Working with Alerts
 
-For every Alert that is used, there should be a gap in between each Alert, then a blank comment. After that, the next Alerts can be used.
+The Five Alerts (Note, Tips, Important, Caution, and Warning) are recognized inside the `[! ]`. This should be able to support multiple lines as long as it is continuing from the previous alerts. Please find the examples below.
 
 * One Alert
 
 ```md
-> [!NOTE] Keyman Facts.
+> [!NOTE] 
+> Keyman Facts.
 
 ```
 
 * More than one
 
 ```md
-> [!TIPS] Keyboard layouts are defined with a clear and easy to understand keyboard grammar.
-                                                                                    // Always leave a line break.
-<!-- Must Include a blank comment between each Alerts -->
-> [!IMPORTANT] Keyman originally created in 1993 to type Lao on Windows.
-
-<!--  -->
-> [!CAUTION] Keyboard layouts are distributed through an open catalog to all major desktop and mobile platforms.
-
-<!-- Last comment -->
-> [!WARNING] This is the last GFMAlerts.
+> [!TIPS] 
+> Keyboard layouts are defined with a clear and easy to understand keyboard grammar.
+                                                                                    
+> [!IMPORTANT] 
+> Keyman originally created in 1993 to type Lao on Windows.
+> Support new lines.
+
+> [!CAUTION] 
+> Keyboard layouts are distributed through an open catalog to all major desktop and mobile platforms.
+> [!WARNING] 
+> This is the last GFMAlerts.
 
 ```
 
 ## Limitation
 
-The Alerts can't recognize new lines, HTML tags, nor handle two lines or more.
+The Alerts can't recognize the inline content and HTML tags. Please stick to the rules/syntax.
 
 ```md
 > [!WARNING] DO NOT do this.
```

Comment:
For the "DO NOT do this" examples, maybe have examples of what happens with inline content and HTML tags.

(and a comment `// this will ....`)